<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>作业帮审核工具</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>1191</width>
     <height>641</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="page_calculate">
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_trangle">
    <widget class="QLineEdit" name="trangle_lineE_1">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>三角形</string>
     </property>
    </widget>
    <widget class="QLabel" name="trangle_lable_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>55</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>底：</string>
     </property>
    </widget>
    <widget class="QLabel" name="trangle_lable_2">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>55</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>高：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="trangle_lineE_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="trangle_lineE_3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>50</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="trangle_lable_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>55</y>
       <width>71</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>三角形面积：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="trangle_lineE_4">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>50</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="trangle_lable_4">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>55</y>
       <width>91</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>平行四边形面积：</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_circle">
    <widget class="QLabel" name="circle_image_1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>20</y>
       <width>54</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="circle_image_2">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>大半径：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="circle_lineE_1">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>小半径：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="circle_lineE_2">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>阴影面积：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="circle_lineE_3">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>140</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="circle_lineE_4">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>170</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>阴影周长：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>110</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>小半径：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="circle_lineE_5">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>80</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>大半径：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="circle_lineE_6">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>140</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>阴影面积：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>170</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>阴影周长：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="circle_lineE_7">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>140</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="circle_lineE_8">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>170</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_6">
     <property name="geometry">
      <rect>
       <x>1145</x>
       <y>10</y>
       <width>20</width>
       <height>621</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_7">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>0</y>
       <width>1150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_8">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>620</y>
       <width>1150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_9">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>20</width>
       <height>191</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_10">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>190</y>
       <width>305</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_11">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>20</width>
       <height>191</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_graphic">
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>外圆内方</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>65</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>圆周长：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="graphic_lineE_1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>65</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>圆面积：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="graphic_lineE_2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>方周长：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="graphic_lineE_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>85</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>90</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>方面积：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="graphic_lineE_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>85</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>112</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>阴影周长：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="graphic_lineE_5">
     <property name="geometry">
      <rect>
       <x>77</x>
       <y>110</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>112</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>阴影面积：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="graphic_lineE_6">
     <property name="geometry">
      <rect>
       <x>195</x>
       <y>110</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>65</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>圆半径：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="graphic_lineE_7">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="garpaic_image_1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>10</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>这是 图片</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>595</x>
       <y>65</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>圆半径：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="graphic_lineE_14">
     <property name="geometry">
      <rect>
       <x>635</x>
       <y>60</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>415</x>
       <y>20</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>外圆内方</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>485</x>
       <y>90</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>方面积：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>90</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>方周长：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="graphic_lineE_10">
     <property name="geometry">
      <rect>
       <x>415</x>
       <y>85</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="graphic_lineE_11">
     <property name="geometry">
      <rect>
       <x>525</x>
       <y>85</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>485</x>
       <y>65</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>圆面积：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>112</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>阴影周长：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="graphic_lineE_12">
     <property name="geometry">
      <rect>
       <x>432</x>
       <y>110</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLabel" name="garpaic_image_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>475</x>
       <y>10</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>这是 图片</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>495</x>
       <y>112</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>阴影面积：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>375</x>
       <y>65</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>圆周长：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="graphic_lineE_9">
     <property name="geometry">
      <rect>
       <x>525</x>
       <y>60</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="graphic_lineE_8">
     <property name="geometry">
      <rect>
       <x>415</x>
       <y>60</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QLineEdit" name="graphic_lineE_13">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>110</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>20</width>
       <height>141</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>140</y>
       <width>1161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>-5</x>
       <y>10</y>
       <width>20</width>
       <height>620</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="Line" name="line_4">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>0</y>
       <width>1150</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="btn_calculate">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>普通计算</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_trangle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>三角形相关</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Circle">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>圆相关</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Graphic">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>图形组合</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
